js/views/PageNav.js

```diff
@@ -0,0 +1,67 @@
+import electron from 'electron';
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+
+const remote = electron.remote;
+
+export default class PageNav extends View {
```

Comment:
Yes, according to Wolf there won't be an appbar and the controls are back in pageNav, so pageNav is going to have to be there from the start.

Once we start implementing the start-up process we can manage how to handle if the app isn't for the full functionality of the pageNav yet, whether it's disabling the address bar and drop-down menus, not displaying them or something else.